src/Html.php

```diff
@@ -1878,7 +1826,10 @@ public static function footer($keepDB = false)
 
         TemplateRenderer::getInstance()->display('layout/parts/page_footer.html.twig', $tpl_vars);
 
-        self::displayDebugInfos();
+        if (!str_starts_with($_SERVER['PHP_SELF'], $CFG_GLPI['root_doc'] . '/install/')) {
+            \Glpi\Debug\Profiler::getInstance()->stopAll();
+            (new Glpi\Debug\Toolbar())->show();
```

Comment:
This produces a `Cannot read properties of undefined (reading 'init')` javascript error when debug mode is inactive due to presence of `window.GLPI.Debug.init({...});` in page source.